### PR TITLE
Add no_log_fix

### PR DESCRIPTION
**Pull request by ashwini-mhatre**
##### SUMMARY

Depends-On: https://github.com/ansible-collections/ansible.netcommon/pull/246

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: plugins/module_utils/network/iosxr/facts/ospfv2/ospfv2.py [hunk 1]
```diff
@@ -147,7 +147,7 @@ def populate_facts(self, connection, ansible_facts, data=None):
         facts = {}
         if current:
             params = utils.validate_config(
```
Comment (NilashishC) [inline plugins/module_utils/network/iosxr/facts/ospfv2/ospfv2.py:149]:
```suggestion
            params = rmmod.validate_config(
```
```suggestion
            params = utils.validate_config(
```

File: plugins/module_utils/network/iosxr/facts/ospfv3/ospfv3.py [hunk 2]
```diff
@@ -145,7 +145,7 @@ def populate_facts(self, connection, ansible_facts, data=None):
         facts = {}
         if current:
             params = utils.validate_config(
```
Comment (NilashishC) [inline plugins/module_utils/network/iosxr/facts/ospfv3/ospfv3.py:147]:
```suggestion
            params = rmmod.validate_config(
```

File: plugins/module_utils/network/iosxr/facts/ospfv2/ospfv2.py (hunk 1 above)
Comment (NilashishC) [inline plugins/module_utils/network/iosxr/facts/ospfv2/ospfv2.py:149]:
```suggestion
            params = rmmod.validate_config(
```

File: plugins/module_utils/network/iosxr/facts/ospfv3/ospfv3.py (hunk 2 above)
Comment (ashwini-mhatre) [reply plugins/module_utils/network/iosxr/facts/ospfv3/ospfv3.py:149]: ack

File: plugins/module_utils/network/iosxr/facts/ospfv2/ospfv2.py (hunk 1 above)
Comment (ashwini-mhatre) [reply plugins/module_utils/network/iosxr/facts/ospfv2/ospfv2.py]: ack

**Comment by ashwini-mhatre** [comment]
recheck

**Review by NilashishC — APPROVED** [review]

**Pull request merged**